homeassistant/components/homekit_controller/climate.py

```diff
@@ -56,15 +89,219 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 
     @callback
     def async_add_service(aid, service):
-        if service["stype"] != "thermostat":
-            return False
         info = {"aid": aid, "iid": service["iid"]}
-        async_add_entities([HomeKitClimateEntity(conn, info)], True)
-        return True
+        if service["stype"] == "thermostat":
```

Comment:
Can you use a dict here like the other entity types? E.g. https://github.com/home-assistant/core/blob/dev/homeassistant/components/homekit_controller/binary_sensor.py#L120